scripts/cuda/test_cuda_detailed.py

```
@@ -0,0 +1,136 @@
+    result = subprocess.run(['nvidia-smi', '--query-gpu=name,driver_version,cuda_version', '--format=csv,noheader'], 
+                          capture_output=True, text=True, timeout=10)
+    if result.returncode == 0:
+        print(f"   GPU: {result.stdout.strip()}")
+    else:
+        print(f"   nvidia-smi error: {result.stderr.strip()}")
+except Exception as e:
+    print(f"   nvidia-smi unavailable: {e}")
+
+# 3. Python CUDA packages
+print("\n3. Python CUDA Packages:")
+import pkg_resources
```

Ruff (0.12.2)

Module level import not at top of file

(E402)